starter-code/css/main.css

```diff
@@ -6,3 +6,152 @@ Green: #00B172
 White: #FFF
 
 */
+body{
+  font-family: 'Montserrat', sans-serif;
+  margin: 0;
+}
+main nav {
```

Comment:
If you want to have a nice navbar with a fixed position, you can try something like this : 

```css
#header {
  position: fixed;
  height: 90px;
  background-color: white;
  width: 100%;
  display: flex;
  box-shadow: 0px 10px 5px -8px rgba(0, 0, 0, 0.75);
}
```